blocks/hero-career/hero-career.js

```diff
@@ -0,0 +1,54 @@
+import {
+  createOptimizedPicture,
+  getMetadata,
+  readBlockConfig,
+} from '../../scripts/lib-franklin.js';
+
+function deleteConfigBlock(block, firstNonCfgEl) {
+  while (block.children.length > 0 && block.children[0] !== firstNonCfgEl) {
+    block.children[0].remove();
+  }
+}
+
+function addTextEl(tag, txt, parent, ...classes) {
+  const newDiv = document.createElement(tag);
+  newDiv.textContent = txt;
+  classes.forEach((c) => newDiv.classList.add(c));
+  parent.appendChild(newDiv);
+}
+
+export default function decorate($block) {
```

Comment:
Does `$` carry a specific meaning here?

Reply:
No, I just started this block originally from another block which had the `$` syntax. I'll remove it.